frontend/integration-tests/views/secrets.view.ts

```diff
@@ -38,7 +38,7 @@ export const addSecretEntryLink = $('.co-create-secret-form__link--add-entry');
 export const removeSecretEntryLink = $$(
   '.co-create-secret-form__link--remove-entry .pf-m-link',
 ).first();
-export const imageSecretForm = $$('.co-create-image-secret__form');
+export const imageSecretForm = $$('.co-m-pane__body-group');
```

Comment:
We probably want a `data-test-id` here. This no longer really identifies the form.